pipelines/SpeciesTreeFromBusco/main.nf

```diff
@@ -725,12 +824,25 @@ workflow {
     prepareBusco(params.busco_proteins)
 
     // Prepare input genomes:
-    genomes = Channel.fromPath("${params.dir}/*.*")
+    if (params.dir != "") {
+        genomes = Channel.fromPath("${params.dir}/*.*", type: 'file')
+    } else {
+        genomes = Channel.of()
+    }
```

Comment:
Would this conditional initialisation of `genomes` and of `genCsvChan` below merit a comment?